wiki/Beatmapping/id.md

```diff
@@ -1,16 +1,56 @@
 
-Tingkat kesulitan umum untuk beatmap adalah:
+Mapper biasanya ingin beatmapnya [ranked](/wiki/Beatmaps/#ranked) dimana terdiri dari [beatmap ranking procedure](/wiki/Beatmap_Ranking_Procedure).
 
--   [*Easy*](ID:Easy_(Difficulty) "wikilink")
--   [*Normal*](ID:Normal "wikilink")
--   [*Hard*](ID:Hard "wikilink")
--   [*Insane*](ID:Insane "wikilink")
+_osu!_ adalah permainan musik game, Musik yang kamu pilih harus memiliki beat yang jelas dan konstan.
+Hindari musik yang memiliki tempo (kecepatan) berubah atau musik lambat sampai kamu mendapatkan pengalaman dan terbiasa dalam membuat beatmap.
 
-Bahkan ada tingkatan khusus pula:
+## Memulai
 
--   [TAG](ID:TAG "wikilink")
+Untuk memulai membuat beatmap baru, silakan pergi ke edit mode kemudian drag-dan-drop audio file ke _osu!_.
+(Pilihan lain, kamu bisa memindahkan file audionya ke dalam folder `Songs`.)
+Ini akan langsung menuju ke dalam layar edit, bersamaan dengan audio yang kamu masukan tadi.
+Musik baru kamu akan berada dibawah daftar musik, biasanya berwarna ungu.
+Jika tidak muncul dalam daftar musik untuk alasan apapun, tekan `F5` untuk me-reset cache beatmap dan musik pasti akan muncul.
+Klik musik, ini akan membuka [beatmap editor](/wiki/beatmap_editor) dan dialog `Song Setup`.
 
-Setiap link memiliki sekumpulan pedoman umum untuk diikuti para [mapper](ID:Mappers "wikilink") agar dapat membuat sebuah [beatmap](ID:Beatmap "wikilink") yang menyenangkan yang sesuai dengan tingkat kesulitannya.
+### Song Setup
+
+_Lihat juga: [Song Setup](/wiki/Song_Setup)_
+
+Ketika akan membuat [mapset](/wiki/mapset) baru, Kamu akan melihat dialog ini sebelum bagian-bagian lain yang terdapat di beatmap editor.
+
+Kamu harus mengisi informasi terlebih dahulu sebelum membuat [difficulty](/wiki/difficulty) lain.
+Ini akan memastikan bahwa metadata konsisten di semua tingkat kesulitan dalam mapset.
+
+Kamu dapat mengubah opsi di tab lain untuk sekarang jika itu yang kamu mau, tapi itu dapat dirubah nanti.
+Klik `OK`, dan kamu akan memulai proses pembuatan beatmap!
+Langkah selanjutnya adalah mengisi timing kedalam beatmap kamu.
+
+### Timing
+
+_Lihat juga: [Timing](/wiki/Timing)_
+
+**Timing itu penting!**
+Beatmap yang memiliki timing yang salah tidak dipertimbangkan untuk ranking.
+Kamu harus sudah yakin timing kamu sudah bagus atau tidak sebelum menambahkan [hit objects](/wiki/hit_objects).
+
+Jika kamu tidak bisa menentukan timing, kontak seorang mod untuk mendapatkan bantuan, atau mengirimkan musik kamu ke [Beatmap Help forum](https://osu.ppy.sh/forum/10).
+Ini akan membantu kamu (dan komunitas modding) dari kesulitan yang ada.
+
+### Compose
+
+_Lihat juga: [Compose](/wiki/Compose)_
+
+Compose tab di dalam beatmap editor adalah tempat menghabiskan sebagian besar waktu kamu untuk pembuatan beatmap.
+Ini adalah tempat kamu meletakaan hit object secara visual dan mengatur beberapa [hit sounds](/wiki/hit_sounds)
+
+### Design
```

Comment:
Change "Design" to "Desain"

Reply:
Should we do `Rancangan`?

Reply:
`Rancangan` is ok but I think `Desain` is more suitable because if we use `Rancangan` it will confuse mapping with storyboarding.

Reply:
actually it's default name of `Design` tab, you can't change this, referred from another languages